app/models/cause.rb

```diff
@@ -0,0 +1,2 @@
+class Cause < ActiveRecord::Base
```

Comment:
Looks like we're missing a relation here.

In our `Donation` model, we say that a `Donation` has one cause, which would imply that `Cause` should belong to a `Donation`.

This doesn't seem to be in line with our ERD, which says that a `Donation` should belong to a `Cause`, and a `Cause` should have many donations.